File: 1011.c
/*
    @autor: Guiventura6;
    @data: 20/01/2022;
    @Problema: Sphere;
*/
#include <stdio.h>
#include <math.h>

#define PI 3.14159;

int main(){
    
    double radius, volume;

    scanf("%lf", &radius);
        
    volume = 4.0 / 3 * pow(radius, 3) * PI;

    printf("VOLUME = %.3lf\n", volume);

    return 0;
}

File: 1011.py
''' 
    Autor: Guiventura6;
    Data: 01/03/2022;
    Problema: Sphere;
    Resume: Program that calculates and shows the volume of a sphere 
    being provided the value of its radius    
'''

def volume(radius):
    pi = 3.14159
    return (4.0/3.0) * pi * radius ** 3.0

radius = int(input())

print('VOLUME = {:.3f}'.format(volume(radius)))
